src/main/java/uk/gov/hmcts/darts/arm/service/impl/UnstructuredToArmProcessorImpl.java

```diff
@@ -221,7 +220,9 @@ private boolean copyRawDataToArm(ExternalObjectDirectoryEntity unstructuredExter
                                      String filename,
                                      ObjectRecordStatusEntity failedStatus) {
         try {
-            if (failedStatus == null || failedStatus.getId().equals(FAILURE_ARM_RAW_DATA_FAILED.getId())) {
+            if (failedStatus == null
+                || FAILURE_ARM_RAW_DATA_FAILED.getId().equals(failedStatus.getId())
+                || ARM_INGESTION.getId().equals(failedStatus.getId())) {
```

Comment:
Does `failedStatus` variable name need to be renamed? This doesn't seem to be a failed one e.g. `FAILURE_ARM_INGESTION_FAILED`?

Reply:
FAILURE_ARM_INGESTION_FAILED isn't being used anymore as that applied when there was just one failure on the arm raw data upload but now it can fail on manifest as well. The other failure FAILURE_ARM_MANIFEST_FILE_FAILED happens after this one has succeeded.